docs-src/python/how-to-use-test-frameworks-in-python.md

```diff
@@ -0,0 +1,12 @@
+---
+id: how-to-use-test-frameworks-in-python
+title: How to use test frameworks in Python
+sidebar_label: Test frameworks
+description: Use Pytest for testing frameworks in Python.
+tags:
+  - developer-guide
+  - python
+  - testing
+---
+
+One recommended framework for testing in Python for the Temporal SDK is [pytest](https://docs.pytest.org/), which can help with fixtures to stand up and tear down test environments, provide useful test discovery, and make it easy to write parametrized tests.
```

Comment:
```suggestion
Temporal recommend [pytest](https://docs.pytest.org/) for testing code in the Python SDK. This library provides fixtures to stand up and tear down test environments, along with useful test discovery and methods to write parametrized tests.
```

Reply:
I know I wrote something similar down below. Again, if it's library or class, please fix. Or ignore it entirely. Up to you.